### ISSUE TITLE
Closing a lightbox that contains a multimedia player doesn't pause in Firefox/Opera

### ISSUE THREAD

**Issue by EricDunsworth** [wet-boew/wet-boew#8582]
We've noticed that if a user attempts to playback a multimedia player that's situated inside a lightbox overlay, closing the overlay will automatically pause the video in some browsers... but not others.

The player only seems to auto-pause when closing a lightbox in IE11. In Firefox (Gecko) and Opera (Blink), the player continues playing after its parent lightbox is closed. This is bad from an accessibility perspective since it causes the user has no means of controlling playback settings while the player is completely off-screen.

Would it be possible to update the media player to make auto-pausing work properly across all modern browsers in this scenario?

I've provided an HTML code sample and reproduction steps below.

**HTML code sample:**
```html
<a class="wb-lbx" href="#testVideo">Test video</a>

<section id="testVideo" class="mfp-hide modal-dialog modal-content overlay-def">
	<header class="modal-header">
		<h2 class="modal-title">Test video</h2>
	</header>
	<div class="modal-body">
		<figure class="wb-mltmd">
			<video poster="https://wet-boew.github.io/v4.0-ci/demos/multimedia/demo/video1-en.jpg" title="Looking for a Job">
				<source type="video/mp4" src="https://video2.servicecanada.gc.ca/video/boew-wet/te-lj-eng.mp4" />
				<track src="https://wet-boew.github.io/v4.0-ci/demos/multimedia/cpts-lg-en.html" kind="captions" data-type="text/html" srclang="en" label="English" />
			</video>
			<figcaption>
				<p>Looking for a Job (<a href="https://wet-boew.github.io/v4.0-ci/demos/multimedia/cpts-lg-en.html">Transcript</a>)</p>
			</figcaption>
		</figure>
	</div>
</section>
```

**Reproduction steps:**
1. Click the "Test video" link.
2. Start playing the video.
3. Close the overlay (either click the "Close" button or click outside the popup).
4. Click the "Test video" link again.

**Expected outcome (IE11):**
After step 3, the video should automatically pause (it's audio should stop). After step 4, the video should be seen as paused.

**Actual outcome (Firefox/Opera):**
After step 3, the video continues playing (its audio can still be heard). After step 4, the video is still playing.

**Comment by juleskuehn** [comment]
Should this be handled in the multimedia or the lightbox plugin? It seems that some dependency would be created between the two in either case?

**Comment by duboisp** [comment]
The ligthbox should be limited to emit an event that is currently closing.
The multimedia player should listen to event that could impact it current state, like listening to a closing event and stop the video if needed.
